src/QAT/qat/purr/compiler/instructions.py

```diff
@@ -40,13 +41,7 @@ def __repr__(self):
         return self.name
 
 
-class Instruction:
-    @staticmethod
-    def deserialize(blob, model: QuantumHardwareModel):
-        return json_loads(blob, model=model)
-
-    def serialize(self):
-        return json_dumps(self)
+class Instruction: ...
```

Comment:
minor: a new line before the `...` ^^

Reply:
@hamidelmaazouz This is how black likes it. I will make #79 formatted the same.

Reply:
then it's all fine :+1:

Reply:
thank you